dace/frontend/python/replacements.py

```diff
@@ -380,24 +380,22 @@ def _sum(sdfg: SDFG, state: SDFGState, a: str, axis=None):
 
 @oprepo.replaces('numpy.max')
 def _max(sdfg: SDFG, state: SDFGState, a: str, axis=None):
-    # HACK: reduce doesn't work if identity isn't specified (at the moment)
     return _reduce(sdfg,
                    state,
                    "lambda x, y: max(x, y)",
                    a,
                    axis=axis,
-                   identity=-9999999999999)
+                   identity=dtypes.min_value(sdfg.arrays[a].dtype))
```

Comment:
I don't think `min_value` does what you think: https://github.com/spcl/dace/pull/252/files#diff-ee5871dec60e3ef36e1051500755f730R232

I do it like this because -max wouldn't work for the int dtypes.

Reply:
(Although it was wrong for `float16`)

Reply:
ok, I missed the branch. good.